composer.json

```diff
@@ -83,7 +83,7 @@
     "league/flysystem-sftp": "1.*",
     "league/fractal": "0.*",
     "litipk/flysystem-fallback-adapter": "0.*",
-    "mschindler83/fints-hbci-php": "1.*",
+    "nemiah/php-fints": "dev-master",
```

Comment:
Unfortunately, there is still no release that supports this, but we might want to at least pin a commit?